src/daos/docker/build-dev.sh

```diff
@@ -51,35 +59,47 @@ function get_s3_dockerfile()
 
 function get_ceph_dockerfile()
 {
-    if [[ $branch == $default_branch ]]; then
+    if [[ $CEPH_BRANCH == $default_branch ]]; then
         echo "Dockerfile"
     else
-        sed "s/add-daos-rgw-sal/${branch}/g; s/daos-single-host/daos-single-dev/g" < Dockerfile > /tmp/Dockerfile.dev.ceph
+        sed "s/add-daos-rgw-sal/${CEPH_BRANCH}/g; s/daos-single-host/daos-single-dev/g" < Dockerfile > /tmp/Dockerfile.dev.ceph
         echo "/tmp/Dockerfile.dev.ceph"
     fi
 }
 
-if [[ $branch == '' ]]; then
+if [[ $CEPH_BRANCH == '' ]]; then
     echo "Failed to identify/use the branch specified"
     exit 1
 fi
 
-daosrocky=`docker images | grep -o "daos-rocky"`
-if [[ ! $daosrocky == 'daos-rocky' ]]; then
-    docker build https://github.com/daos-stack/daos.git#master -f utils/docker/Dockerfile.el.8 -t daos-rocky
+function build_daos()
+{
+    set +e
+    daosrocky=`docker images | grep -o "daos-rocky"`
+    set -e
+    if [[ ! $daosrocky == 'daos-rocky' ]]; then
+        docker build "https://github.com/daos-stack/daos.git#${DAOS_BRANCH}" -f utils/docker/Dockerfile.el.8 -t daos-rocky
+        if [[ ! $? == 0 ]]; then exit 1; fi
+    fi
+    pushd daos
+    docker build . -t "daos-single-$image_type"
```

Comment:
Where is `$image_type` defined?

Reply:
just a few lines down
`image_type="host"
if [[ ! $CEPH_BRANCH == $default_branch ]]; then
    image_type="dev"
fi
`